cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/IdentifierPass.kt

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
Even if we don't need this pass any more for the order evaluation, I'd keep it just in case other projects make use of it. We should set deprecate it though

Reply:
I've re-added the pass